.github/workflows/pyoxidizer.yml

```diff
@@ -0,0 +1,178 @@
+name: Build PyOxidizer Binary
+on:
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+
+jobs:
+  build-linux-gnu:
+    name: Build Linux (GNU)
+    runs-on: ubuntu-latest
+    container: quay.io/pypa/manylinux2014_x86_64
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cache/pyoxidizer/
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+          key: linux-cache
+
+      - name: Install dependencies
+        run: /opt/python/cp39-cp39/bin/pip install pyoxidizer dissect.target
```

Comment:
This runs the risk of installing dissect dev packages. It is better to install the `dissect` package.

However as this workflow triggers on tags, and acquire is usually tagged before a new dissect package is tagged&released, this will still not work. I think it is best to make this a manual workflow where a tag can be given as parameter.

Reply:
That's only when you add `--pre` right? This just installs the latest stable `dissect.target`.

Reply:
Yes, but it will only install the stable version of dissect.target not its dissect.* dependencies.